Roihu updates to Docs CSC

rkronberg opened the pull request:
# Development branch for Roihu updates to Docs CSC

This is the development branch for Roihu-related documentation. **Shared with e.g. pilot users and to be merged into production at GA**.

Preview: https://csc-guide-preview.2.rahtiapp.fi/origin/roihu/

## For contributors

1. Please create a branch **from this** development branch (`roihu`)
2. Create/update content
3. Merge into **this branch** (`roihu`)

This way we keep the new updates separate from production docs until Roihu is available for everyone.

### On the command-line

```
git clone git@github.com:CSCfi/csc-user-guide.git
cd csc-user-guide
git checkout roihu
git checkout -b <new branch name>
# do updates
git add .
git commit -m "Commit message"
git push origin <new branch name>
```

### In GitHub

Go to the Pull requests section here in docs Github. Create a new pull request from your branch, and change the base branch into which you want to merge your branch from `master` to `roihu`, see image.

After you confirm your changes, you can merge the pull request into `roihu`. Remember to check also the preview page for your branch, which will be created after a few minutes in https://csc-guide-preview.2.rahtiapp.fi/origin/<name-of-your-branch>/.
Merging does not require approval or admin rights since `roihu` branch is not protected. 
Nevertheless, **ask for an approving review** before merging your changes into `roihu`.

<img width="951" height="290" alt="image" src="https://github.com/user-attachments/assets/4093d857-47c7-47f7-8415-d2b8dce33e95" />

rkronberg committed:
Roihu updates to Docs CSC

rkronberg committed:
Merge branch 'master' into roihu

rkronberg committed:
Merge branch 'master' into roihu

rkronberg committed:
Merge branch 'master' into roihu

rkronberg committed:
Merge branch 'master' into roihu

rkronberg committed:
Merge branch 'master' into roihu

rkronberg committed:
Merge branch 'master' into roihu

rkronberg committed:
Merge branch 'master' into roihu

rkronberg committed:
Merge branch 'master' into roihu

rkronberg committed:
Roihu migration guide (#2671)

* Roihu migration guide

* SSH CA (#2810)

* SSH CA

* headers

* add mycsc instructions

* add instructions

* fix linkl

* fix link

* extra step

* instructions

* SSH agent

* computing index page

* fix link

* use snippets

* roihu fingerprints for pilot phase

* updates

* authentication agent for windows

* agent forwarding

* document python version of cert helper

* quickstart guide

* add some link

* Add graphics how SSH login works

Just a try to add something graphical. Would need at least some text around etc, but remove if not needed.

* implement suggestions

* fix links

* fix links

* move diagram up

* migration guide: general notes

* adjust auto-approved limits

* updates

* update ssh ca instructions

* some updates

* data migration guide v1

* fix links

* SSH CA updates

* more ssh ca updates

* more ssh ca updates

* update rsync, scp, tar-ssh

* filezilla

* winscp

---------

Co-authored-by: Kylli Ek <kylli.ek@csc.fi>

rkronberg committed:
update banner

galfthan commented on docs/computing/connecting/ssh-keys.md:
```diff
@@ -103,7 +110,180 @@ cat /var/lib/acco/sshkeys/${USER}/${USER}.pub
+        6. Optional, but **strongly recommended**:
+           [Install WinSCP](https://winscp.net/eng/docs/installation) and
+           [start the Pageant authentication agent](https://the.earth.li/~sgtatham/putty/0.83/htmldoc/Chapter9.html#pageant)
+           that comes bundled with
```
Pageant can also be installed as part of WinSCP:
https://winscp.net/eng/docs/ui_pageant#obtaining_and_starting_pageant

rkronberg committed:
Merge branch 'master' into roihu

rkronberg committed:
test Roihu apps tags

rkronberg committed:
Merge branch 'roihu' of github.com:CSCfi/csc-user-guide into roihu

rkronberg committed:
fix bolding

rkronberg committed:
typo

rkronberg committed:
improve based on comments

rkronberg committed:
Link to public cert helper tool

rkronberg committed:
clone

rkronberg committed:
save link as

mvsjober committed:
Merge branch 'master' into roihu

mvsjober committed:
Added PyTorch and vLLM for Roihu

mvsjober committed:
Fixed vLLM link

mvsjober committed:
Fixed errors in vLLM.

rkronberg committed:
fix link

rkronberg committed:
Gromacs on Roihu (#2921)

rkronberg committed:
Merge branch 'master' into roihu

rkronberg committed:
CP2K on Roihu (#2922)

rkronberg committed:
Maestro on Roihu (#2923)

* Maestro on Roihu

* rephrase

leopekkas committed:
Add Argos error information for pilot projects

leopekkas committed:
Add preliminary information on running, fix links to be dynamic vs. static

leopekkas committed:
Add links to Roihu on the computing index page

leopekkas committed:
Add sections for installing and running jobs in Roihu

leopekkas committed:
Fix list formatting

leopekkas committed:
Remove specific project id from example Slurm script

leopekkas committed:
Fix warning section formatting

leopekkas committed:
Add link to relevant docs pages on SSH

niemilau committed:
Roihu docs on compiling (#2938)

* Roihu compilation instructions

* Grammar fixes

Co-authored-by: Martti Louhivuori <martti.louhivuori@csc.fi>

* - Updated nvcc instructions
- Use Roihu-CPU and Roihu-GPU names when referring to partitions
- Small formatting changes

* Wrote basic compilation instructions for Roihu-CPU.

Some TODOs left still

* Added instructions for compiling with MPI on Roihu-GPU, plus general cleanup

* Added instructions for using provided HPC libraries on Roihu

* Fixed link to CUDA section

* Fixed Roihu-CPU sub-sectioning

* Apply suggestions from code review

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

* Raise roihu section above mahti/puhti

* wording

---------

Co-authored-by: leopekkas <leopekkas@gmail.com>
Co-authored-by: Martti Louhivuori <martti.louhivuori@csc.fi>
Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

leopekkas committed:
Merge pull request #2934 from CSCfi/roihu-computing

Add Computing index link to compiling on Roihu

leopekkas committed:
Merge pull request #2932 from CSCfi/roihu-quickstart

Expand Roihu's getting started tutorial

Iidahak committed:
update python docs for Roihu

Iidahak committed:
update python docs for Roihu

Iidahak committed:
update python docs

Iidahak committed:
update python docs

Iidahak committed:
add vllm to module list

Iidahak committed:
add vllm to module list

Iidahak committed:
fix link to geoconda.md

Iidahak committed:
fix link to geoconda.md

Iidahak committed:
update python docs

Iidahak committed:
update python docs

Iidahak committed:
fix python usage guide

Iidahak committed:
fix python usage guide

Iidahak committed:
update python docs

Iidahak committed:
update python docs

Iidahak committed:
fix link

Iidahak committed:
fix link

leopekkas committed:
Fix broken links in quickstart guide (#2942)

* Fix broken links in quickstart guide
* Fix Python guide and application list links
* Fix link for creating Slurm scripts

leopekkas committed:
WIP Roihu quickstart guide link fixes (#2943)

* Fix broken links and link formatting in quickstart guide

ktiits committed:
Add python-geo (#2944)

* Add python-geo

* remove geoplot from the list

* Specify Roihu-CPU

Iidahak committed:
Merge pull request #2941 from CSCfi/roihu-python-data

Update Python docs for Roihu

trossi committed:
Roihu docs on affinity (#2936)

* Reorder

* Add OMP_PROC_BIND

* Move affinity section to a separate tutorial

* Apply suggestions from review

---------

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

trossi committed:
Roihu docs on affinity (#2936)

* Reorder

* Add OMP_PROC_BIND

* Move affinity section to a separate tutorial

* Apply suggestions from review

---------

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

leopekkas committed:
Update roihu disk quotas and formatting in tutorials

leopekkas committed:
Update roihu disk quotas and formatting in tutorials

leopekkas committed:
Fix formatting and link to the correct CPU compiling section

leopekkas committed:
Fix formatting and link to the correct CPU compiling section

Iidahak committed:
Update python.md

Add information of the separate module trees to python.md

Iidahak committed:
Update python.md

Add information of the separate module trees to python.md

leopekkas committed:
Fix formatting, link to roihu applications

leopekkas committed:
Fix formatting, link to roihu applications

Commit:
roihu base containers

Commit:
roihu base containers

mvsjober committed:
Merge pull request #2945 from CSCfi/roihu-tutorials

WIP Roihu tutorials

Commit:
base containers

Commit:
base containers

mvsjober commented on docs/computing/connecting/ssh-keys.md:
```diff
@@ -103,7 +110,189 @@ cat /var/lib/acco/sshkeys/${USER}/${USER}.pub
+**SSH certificates are valid for 24 hours at a time**. Once your certificate
+expires, a new one must be signed following either of the processes below.
+
+### Option 1: Certificate helper tool (recommended)
```
I'm a bit hesitant if the Certificate helper tool should be option 1 and marked as "recommended". It's great to have a helper script, but it might be a bit of a big hurdle for a new user to download some Python script first. I would instead first do it the MyCSC way, and then if you find that too awkward, we have a nice helper tool to make it more automated.

leopekkas replied:
Suggestion applied here:

https://github.com/CSCfi/csc-user-guide/pull/2597/changes/9b5a1e2af6840c9486779dd12f64b426f2c7f7bd

mvsjober commented on docs/computing/connecting/ssh-keys.md:
```diff
@@ -103,7 +110,189 @@ cat /var/lib/acco/sshkeys/${USER}/${USER}.pub
+
+1. Ensure that you have Python installed on your computer.
+    - Instructions are available in the
+      [Python Beginners Guide](https://wiki.python.org/moin/BeginnersGuide/Download).
```
Python Beginners Guide - looking at the link https://wiki.python.org/moin/BeginnersGuide/Download it seems to be archived soon/deprecated, maybe not the best Python link?

leopekkas replied:
Updated to https://www.python.org/downloads/: https://github.com/CSCfi/csc-user-guide/pull/2597/commits/01b2cbd680c94368d85cccbe3aa8a899b8480e82

mvsjober commented on docs/support/tutorials/roihu.md:
```diff
@@ -0,0 +1,244 @@
+
+1. Check if the software is already available:
+     - [List of pre-installed applications](../../apps/by_availability.md#roihu)
+     - `module spider <module name>`
```
```suggestion
Before installing anything check if the software is already available:
     - [List of pre-installed applications](../../apps/by_availability.md#roihu)
     - `module spider <software name>`
```

Numbered list with only one item looks weird.

mvsjober commented on docs/computing/compiling-roihu.md:
```diff
@@ -0,0 +1,218 @@
+    - Roihu-CPU nodes use AMD (x86) processors
+    - Roihu-GPU nodes use NVIDIA Grace (ARM) processors
+
+    Binaries compiled for one architecture are generally not usable on the other.
```
```suggestion
    Binaries compiled for one architecture are not usable on the other.
```

I guess they are never usable on the other so we can remove "generally". (Except maybe for Java bytecode and such?)

mvsjober commented on docs/support/tutorials/roihu.md:
```diff
@@ -0,0 +1,244 @@
+In most cases, ready-made Docker containers can be easily converted into an Apptainer image.
+Another option is to build your own container from scratch. 
+You can build containers on top of Roihu base containers which have the same software stack as is available via the module system natively.
+Base container are built on top of Rockylinux 9.
```
```suggestion
Base container are built on top of Rocky Linux 9.
```

mvsjober commented on docs/support/tutorials/roihu.md:
````diff
@@ -0,0 +1,244 @@
+        exec "$@"
+    ```
+
+    When building the containers, set you cache directory to temporary directory to avoid filling you home directory quota.
````
```suggestion
    When building containers, set the Apptainer cache directory to `$TMPDIR` to avoid filling your home directory quota.
```

Iidahak committed:
update python.md

Iidahak committed:
update python.md

Iidahak committed:
update python.md

Iidahak committed:
update python.md

Iidahak committed:
Merge pull request #2947 from CSCfi/roihu-python

Update python.md

mvsjober commented on docs/support/tutorials/roihu.md:
````diff
@@ -0,0 +1,244 @@
+    apptainer run --cleanenv run mycmd
+    ```
+
+More details on working with containers in CSC's computing environment can be found from the links below:
````
Should we also mention the machine learning base and PyTorch base containers here as they have a slightly different approach?

leopekkas replied:
To discuss with @jaantollander

mvsjober commented on docs/support/tutorials/roihu.md:
```diff
@@ -0,0 +1,244 @@
+#SBATCH --ntasks=1
+#SBATCH --time=DD:HH:MM
+
+srun --argos=no <your-executable>
```
Can't you also have it as an SBATCH option?

```
#SBATCH --argos=no
```

mvsjober commented on docs/support/tutorials/roihu-data.md:
```diff
@@ -0,0 +1,452 @@
+* Once you have identified the data you need to transfer, check that it
+  fits within the default disk quotas on Roihu:
+
+    | Disk area | Path                  | Default size | Max. size [^1]      | Default file number limit | Max. file number limit [^1] |
```
```suggestion
    | Disk area | Path                  | Default size | Max. size      | Default file number limit | Max. file number limit |
```

There are no longer any values in parenthesis so we should remove the footnote.

mvsjober commented on docs/support/tutorials/roihu-data.md:
```diff
@@ -0,0 +1,452 @@
+    | ProjData  | `/projdata/<project>` | 0 GiB        | case-by-case        | 0                         | case-by-case                |
+    | Scratch   | `/scratch/<project>`  | 250 GiB      | 100 TiB             | 500k                      | 10M                         |
+
+    [^1]: Values in parentheses indicate automatically approved limits.
```
```suggestion
```

mvsjober commented on docs/support/tutorials/roihu.md:
```diff
@@ -0,0 +1,244 @@
+
+See the relevant documentation below for detailed information:
+
+1. [Available batch job partitions](../../computing/running/batch-job-partitions.md)
```
The page linked to here does not actually have any Roihu partitions!

leopekkas replied:
Updated Roihu Slurm docs currently under:
https://github.com/CSCfi/csc-user-guide/pull/2939 , and
https://github.com/CSCfi/csc-user-guide/pull/2894

Should be merged ASAP

leopekkas replied:
Updated here: https://github.com/CSCfi/csc-user-guide/pull/2597/commits/631ad78d03d1c450610760a076e8e2731a91f397

mvsjober committed:
Changed Roihu to Roihu-GPU for PyTorch, vLLM

mvsjober committed:
Changed Roihu to Roihu-GPU for PyTorch, vLLM

Commit:
update gpu base container version

Commit:
update gpu base container version

kkmattil committed:
Create allas-in-roihu.md

Instructions for Allas and Lumi-O configuration in Roihu

kkmattil committed:
Create allas-in-roihu.md

Instructions for Allas and Lumi-O configuration in Roihu

Commit:
add --nv flag

Commit:
add --nv flag

Commit:
Merge branch 'roihu' of github.com:CSCfi/csc-user-guide into roihu

kkmattil committed:
Update allas-in-roihu.md

kkmattil committed:
Update allas-in-roihu.md

kkmattil committed:
Update index.md

Link to object storage instructions added

kkmattil committed:
Update index.md

Link to object storage instructions added

kkmattil committed:
Update index.md

kkmattil committed:
Update index.md

kkmattil committed:
Update allas-in-roihu.md

kkmattil committed:
Update allas-in-roihu.md

kkmattil committed:
Update allas-in-roihu.md

kkmattil committed:
Update allas-in-roihu.md

kkmattil committed:
Update allas-in-roihu.md

kkmattil committed:
Update allas-in-roihu.md

kkmattil committed:
Update allas-in-roihu.md

kkmattil committed:
Update allas-in-roihu.md

leopekkas committed:
Apply suggestions from code review

Co-authored-by: Mats Sjöberg <mats.sjoberg@csc.fi>

leopekkas committed:
Apply suggestions from code review

Co-authored-by: Mats Sjöberg <mats.sjoberg@csc.fi>

leopekkas committed:
Add suggestion from review

Co-authored-by: Mats Sjöberg <mats.sjoberg@csc.fi>

leopekkas committed:
Add suggestion from review

Co-authored-by: Mats Sjöberg <mats.sjoberg@csc.fi>

leopekkas committed:
Apply suggestion from review

leopekkas committed:
Apply suggestion from review

leopekkas committed:
Fix argos option that breaks automated tests

leopekkas committed:
Fix argos option that breaks automated tests

ktiits committed:
Roihu ssh ca from windows (#2949)

* fix: advanced level (#2925)

* Urgent announcement feature (#2927)

* Urgent announcement feature

* Enable internal links

Edit content, adjust margins

* Swap url to lue tutorial (#2928)

small change for explanation and rather link to `lue` instructions than pre-cleaning tips

* BeeGFS Pouta tutorial (#2924)

* BeeGFS Pouta tutorial

* Add Benchmark and metadata migration

---------

Co-authored-by: Joona Tolonen <54183803+tolonenj@users.noreply.github.com>

* Update strong-identification.md (#2930)

clarified a few points

Co-authored-by: Joonas Somero <50655931+joonas-somero@users.noreply.github.com>

* Monitor Pukki DBaaS instance sizes from a Rahti CronJob using application credentials (#2926)

* tutorial: Monitor Pukki DBaaS instance sizes from a Rahti CronJob using application credentials

* fix broken link

* fix: polish and add links

* fix bullet points

* fix link

* fix bullet points

* add clone rule

* add clone rule

* Ac conv (#2935)

* Create sd-connect-conversion-tool.md

* Updates

* Update sd-connect-conversion-tool-ui.md

* Update sd-connect-conversion-tool-ui.md

* Update sd-connect-conversion-tool-ui.md

* Update sd-connect-conversion-tool-ui.md

* Navigation

* Update sd-connect-conversion.md

* Fixes

* Fixes

* Fixes

---------

Co-authored-by: Joonas Somero <50655931+joonas-somero@users.noreply.github.com>

* Nextcloud rahti (#2931)

* Tutorial: Nextcloud on Rahti

* Update docs/cloud/rahti/tutorials/nextcloud.md

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* Update docs/cloud/rahti/tutorials/nextcloud.md

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* Update docs/cloud/rahti/tutorials/nextcloud.md

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* Update docs/cloud/rahti/tutorials/nextcloud.md

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* Update docs/cloud/rahti/tutorials/nextcloud.md

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* Update docs/cloud/rahti/tutorials/nextcloud.md

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* How to install kustomize

* Fix internal links

---------

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* Add volume deletion instructions to documentation (#2940)

Co-authored-by: Tristan <74349933+trispera@users.noreply.github.com>

* add lastools 2026 (#2946)

* add lastools 2026

* Update docs/apps/lastools.md

Co-authored-by: EetuHuuskoCSC <116141296+EetuHuuskoCSC@users.noreply.github.com>

---------

Co-authored-by: EetuHuuskoCSC <116141296+EetuHuuskoCSC@users.noreply.github.com>

* Fix broken external links (#2929)

* some rewriting to Windows connecting with CA

* styling fixes

* styling, links

* fix links

* fix links

* Revert "Merge branch 'master' into roihu-ssh-ca"

This reverts commit f2186d5b4eec606618cd8bce8facd1caefabab46, reversing
changes made to 59481e63db5b381eefc54c84d67666e369c7261b.

---------

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>
Co-authored-by: Joonas Somero <50655931+joonas-somero@users.noreply.github.com>
Co-authored-by: attesillanpaa <atte.sillanpaa@csc.fi>
Co-authored-by: Tristan <74349933+trispera@users.noreply.github.com>
Co-authored-by: Joona Tolonen <54183803+tolonenj@users.noreply.github.com>
Co-authored-by: Johanna Baldrighi <48281666+jotaskin@users.noreply.github.com>
Co-authored-by: Aino-Riikka Corell <101800110+ainoc@users.noreply.github.com>
Co-authored-by: EetuHuuskoCSC <116141296+EetuHuuskoCSC@users.noreply.github.com>

ktiits committed:
Roihu ssh ca from windows (#2949)

* fix: advanced level (#2925)

* Urgent announcement feature (#2927)

* Urgent announcement feature

* Enable internal links

Edit content, adjust margins

* Swap url to lue tutorial (#2928)

small change for explanation and rather link to `lue` instructions than pre-cleaning tips

* BeeGFS Pouta tutorial (#2924)

* BeeGFS Pouta tutorial

* Add Benchmark and metadata migration

---------

Co-authored-by: Joona Tolonen <54183803+tolonenj@users.noreply.github.com>

* Update strong-identification.md (#2930)

clarified a few points

Co-authored-by: Joonas Somero <50655931+joonas-somero@users.noreply.github.com>

* Monitor Pukki DBaaS instance sizes from a Rahti CronJob using application credentials (#2926)

* tutorial: Monitor Pukki DBaaS instance sizes from a Rahti CronJob using application credentials

* fix broken link

* fix: polish and add links

* fix bullet points

* fix link

* fix bullet points

* add clone rule

* add clone rule

* Ac conv (#2935)

* Create sd-connect-conversion-tool.md

* Updates

* Update sd-connect-conversion-tool-ui.md

* Update sd-connect-conversion-tool-ui.md

* Update sd-connect-conversion-tool-ui.md

* Update sd-connect-conversion-tool-ui.md

* Navigation

* Update sd-connect-conversion.md

* Fixes

* Fixes

* Fixes

---------

Co-authored-by: Joonas Somero <50655931+joonas-somero@users.noreply.github.com>

* Nextcloud rahti (#2931)

* Tutorial: Nextcloud on Rahti

* Update docs/cloud/rahti/tutorials/nextcloud.md

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* Update docs/cloud/rahti/tutorials/nextcloud.md

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* Update docs/cloud/rahti/tutorials/nextcloud.md

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* Update docs/cloud/rahti/tutorials/nextcloud.md

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* Update docs/cloud/rahti/tutorials/nextcloud.md

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* Update docs/cloud/rahti/tutorials/nextcloud.md

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* How to install kustomize

* Fix internal links

---------

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>

* Add volume deletion instructions to documentation (#2940)

Co-authored-by: Tristan <74349933+trispera@users.noreply.github.com>

* add lastools 2026 (#2946)

* add lastools 2026

* Update docs/apps/lastools.md

Co-authored-by: EetuHuuskoCSC <116141296+EetuHuuskoCSC@users.noreply.github.com>

---------

Co-authored-by: EetuHuuskoCSC <116141296+EetuHuuskoCSC@users.noreply.github.com>

* Fix broken external links (#2929)

* some rewriting to Windows connecting with CA

* styling fixes

* styling, links

* fix links

* fix links

* Revert "Merge branch 'master' into roihu-ssh-ca"

This reverts commit f2186d5b4eec606618cd8bce8facd1caefabab46, reversing
changes made to 59481e63db5b381eefc54c84d67666e369c7261b.

---------

Co-authored-by: Dean Ruina <81315494+DeRuina@users.noreply.github.com>
Co-authored-by: Joonas Somero <50655931+joonas-somero@users.noreply.github.com>
Co-authored-by: attesillanpaa <atte.sillanpaa@csc.fi>
Co-authored-by: Tristan <74349933+trispera@users.noreply.github.com>
Co-authored-by: Joona Tolonen <54183803+tolonenj@users.noreply.github.com>
Co-authored-by: Johanna Baldrighi <48281666+jotaskin@users.noreply.github.com>
Co-authored-by: Aino-Riikka Corell <101800110+ainoc@users.noreply.github.com>
Co-authored-by: EetuHuuskoCSC <116141296+EetuHuuskoCSC@users.noreply.github.com>

Commit:
removed --cleanenv

Commit:
removed --cleanenv

jlento committed:
Add files via upload

jlento committed:
Add files via upload

jlento committed:
Update user-spack.md

Added pointer where corresponding Roihu documentation can be found.

jlento committed:
Update user-spack.md

Added pointer where corresponding Roihu documentation can be found.

jlento committed:
Update roihu-user-spack.md

Added shell quotation to make html rendering look more coherent

jlento committed:
Update roihu-user-spack.md

Added shell quotation to make html rendering look more coherent

jlento committed:
Update roihu-user-spack.md

Testing html conversion

jlento committed:
Update roihu-user-spack.md

Testing html conversion

jlento committed:
Update roihu-user-spack.md

jlento committed:
Update roihu-user-spack.md

kkmattil committed:
Update introduction.md

Roihu updates

kkmattil committed:
Update introduction.md

Roihu updates

Commit:
fix typo

Commit:
fix typo

samiilvonen committed:
Fixed typos

samiilvonen committed:
Fixed typos

samiilvonen committed:
Fixed typos

jlento committed:
Update roihu-user-spack.md

One more prompt removed...

jlento committed:
Update roihu-user-spack.md

One more prompt removed...

leopekkas committed:
Fix formatting in Roihu tutorial

leopekkas committed:
Fix formatting in Roihu tutorial

leopekkas committed:
Apply suggestion on SSH-CA instructions, prioritize the MyCSC route

leopekkas committed:
Apply suggestion on SSH-CA instructions, prioritize the MyCSC route

leopekkas committed:
Add up-to-date Python link/fix typo

leopekkas committed:
Add up-to-date Python link/fix typo

leopekkas committed:
Fix dynamic links to old SSH headers

leopekkas committed:
Fix dynamic links to old SSH headers

trossi committed:
Update Roihu affinity tutorial

Simplify the Roihu affinity tutorial by using the csc-print-affinity tool

trossi committed:
Update Roihu affinity tutorial

Simplify the Roihu affinity tutorial by using the csc-print-affinity tool

samiilvonen committed:
Merge pull request #2953 from CSCfi/roihu-user-spack-cherries

Roihu user spack cherry picked changes

galfthan committed:
Minor fixes to SSH instructions (#2956)

* Minor fixes to SSH instructions

* Small fixes for readability and CI/CD

* Internal link fixes for CI/CD

---------

Co-authored-by: leopekkas <leopekkas@gmail.com>

galfthan committed:
Minor fixes to SSH instructions (#2956)

* Minor fixes to SSH instructions

* Small fixes for readability and CI/CD

* Internal link fixes for CI/CD

---------

Co-authored-by: leopekkas <leopekkas@gmail.com>

Commit:
Updated Running jobs -> Getting started

Commit:
Updated Running jobs -> Getting started

sjsillan committed:
Merge pull request #2959 from CSCfi/roihu-computing-running-getting-started

Updated Running jobs - Getting started

leopekkas committed:
WIP Supercomputer storage/disk sections (#2937)

* Add updated disk quotas on Roihu

* Fix the formatting on tabbed content

* Update disk quotas on Roihu

* Add Roihu local storage details

* Fix header and formatting

* Fix header to fix an error on a broken link

* Add section about projdata into disk.md

* Fix typo, formatting

* Move Roihu storage to a separate page for clarity and easier management

* Roll back Mahti/Puhti disk examples to prior values

* Update roihu-disk.md

- Minor fixes
- Marked `lue` tool references with "not implemented"
- Section "Temporary local disk areas" needs a review and rewrite

* Fix link that breaks CI/CD

* Added roihu lustre specs to lustre page

* roihu scratch performance values added

* burst buffer section added

* typo fix

* fix warning box not rendering

* Add single line fix to retrigger rahti build

---------

Co-authored-by: Juha Lento <jlento@users.noreply.github.com>
Co-authored-by: Henry Barton <henry.barton@csc.fi>

leopekkas committed:
WIP Supercomputer storage/disk sections (#2937)

* Add updated disk quotas on Roihu

* Fix the formatting on tabbed content

* Update disk quotas on Roihu

* Add Roihu local storage details

* Fix header and formatting

* Fix header to fix an error on a broken link

* Add section about projdata into disk.md

* Fix typo, formatting

* Move Roihu storage to a separate page for clarity and easier management

* Roll back Mahti/Puhti disk examples to prior values

* Update roihu-disk.md

- Minor fixes
- Marked `lue` tool references with "not implemented"
- Section "Temporary local disk areas" needs a review and rewrite

* Fix link that breaks CI/CD

* Added roihu lustre specs to lustre page

* roihu scratch performance values added

* burst buffer section added

* typo fix

* fix warning box not rendering

* Add single line fix to retrigger rahti build

---------

Co-authored-by: Juha Lento <jlento@users.noreply.github.com>
Co-authored-by: Henry Barton <henry.barton@csc.fi>

trossi committed:
Roihu docs for nvhpc (#2954)

* Add compilation instructions for nvhpc

* Add more details about compiler environments

* Add general MPI+CUDA compilation instructions

---------

Co-authored-by: Sami Ilvonen <sami.ilvonen@csc.fi>
Co-authored-by: leopekkas <leopekkas@gmail.com>

trossi committed:
Roihu docs for nvhpc (#2954)

* Add compilation instructions for nvhpc

* Add more details about compiler environments

* Add general MPI+CUDA compilation instructions

---------

Co-authored-by: Sami Ilvonen <sami.ilvonen@csc.fi>
Co-authored-by: leopekkas <leopekkas@gmail.com>

leopekkas committed:
Point to roihu-disk from compilation instructions

leopekkas committed:
Point to roihu-disk from compilation instructions

rkronberg committed:
WIP: Roihu Slurm (#2894)

* Roihu Slurm

* update values

* Add links to Roihu Slurm under running/

* Roihu job scripts (#2939)

* Add example job scripts

* Ensure using single node

* Add docs on batch job scripts

* Add information about Roihu temporary storage

* Add instructions for GPUs

* Add example job scripts for Roihu-GPU

* Add vizinteractive to valid partition names

* Add --cpus-per-task for pure MPI jobs

* Clarify GPU scripts

* Update memory amounts

* Add example on SBF

* Change example mem-per-cpu to 1000M

* Add note about nodes on hugemem partitions

* Add links to technical details

* Remove $ to ease copying

* Clarify --hint=nomultithread

* Add example pilot Slurm scripts

* Add pilot partitions to partition test checklist

* Fix pilot partition examples, include OMP settings

* Add current, sensible --mem options

---------

Co-authored-by: leopekkas <leopekkas@gmail.com>
Co-authored-by: Tuomas Rossi <tuomas.rossi@csc.fi>
Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

rkronberg committed:
WIP: Roihu Slurm (#2894)

* Roihu Slurm

* update values

* Add links to Roihu Slurm under running/

* Roihu job scripts (#2939)

* Add example job scripts

* Ensure using single node

* Add docs on batch job scripts

* Add information about Roihu temporary storage

* Add instructions for GPUs

* Add example job scripts for Roihu-GPU

* Add vizinteractive to valid partition names

* Add --cpus-per-task for pure MPI jobs

* Clarify GPU scripts

* Update memory amounts

* Add example on SBF

* Change example mem-per-cpu to 1000M

* Add note about nodes on hugemem partitions

* Add links to technical details

* Remove $ to ease copying

* Clarify --hint=nomultithread

* Add example pilot Slurm scripts

* Add pilot partitions to partition test checklist

* Fix pilot partition examples, include OMP settings

* Add current, sensible --mem options

---------

Co-authored-by: leopekkas <leopekkas@gmail.com>
Co-authored-by: Tuomas Rossi <tuomas.rossi@csc.fi>
Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

leopekkas committed:
Add Roihu Slurm links to the Getting started tutorial

leopekkas committed:
Add Roihu Slurm links to the Getting started tutorial

Commit:
added roihu info and updated examples

Commit:
added roihu info and updated examples

Commit:
link fix

Commit:
link fix

Commit:
available base containers

Commit:
available base containers

Commit:
typo and phrasing

Commit:
typo and phrasing

Commit:
rephrase

Commit:
rephrase

henryjuho committed:
Roihu sinteractive warning (#2969)

* added sinteractive missing to known issues

* added warning to sinteractive docs

* triggering rebuild

* Update docs/computing/running/interactive-usage.md

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

* Update docs/support/tutorials/roihu.md

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

* Fx typo from suggestion

---------

Co-authored-by: Henry Barton <henry.barton@csc.fi>
Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>
Co-authored-by: leopekkas <leopekkas@gmail.com>

henryjuho committed:
Roihu sinteractive warning (#2969)

* added sinteractive missing to known issues

* added warning to sinteractive docs

* triggering rebuild

* Update docs/computing/running/interactive-usage.md

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

* Update docs/support/tutorials/roihu.md

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

* Fx typo from suggestion

---------

Co-authored-by: Henry Barton <henry.barton@csc.fi>
Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>
Co-authored-by: leopekkas <leopekkas@gmail.com>

henryjuho committed:
Merge pull request #2971 from CSCfi/roihu-modules

added roihu info and updated examples

Commit:
removed sinteractive missing warnings

Commit:
removed sinteractive missing warnings

Commit:
info on roihu interactive parititons

Commit:
info on roihu interactive parititons

jussienko committed:
Document how to connect to compute node

jussienko committed:
Document how to connect to compute node

Commit:
upadated example commands

Commit:
upadated example commands

Commit:
typos

Commit:
typos

henryjuho committed:
shared node sbof release date (#2973)

* added expected date for sbof on shared nodes

* typo fix

* made note an info box

---------

Co-authored-by: Henry Barton <henry.barton@csc.fi>

henryjuho committed:
shared node sbof release date (#2973)

* added expected date for sbof on shared nodes

* typo fix

* made note an info box

---------

Co-authored-by: Henry Barton <henry.barton@csc.fi>

henryjuho committed:
Update docs/computing/running/interactive-usage.md

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

henryjuho committed:
Update docs/computing/running/interactive-usage.md

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

henryjuho committed:
Update docs/computing/running/interactive-usage.md

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

henryjuho committed:
Update docs/computing/running/interactive-usage.md

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

Commit:
box formatting

Commit:
box formatting

henryjuho committed:
Merge pull request #2974 from CSCfi/roihu-sinteractive-added

Roihu sinteractive added

trossi committed:
Roihu HPC libraries and compiling (#2972)

* Remove outdated warning

* Update list of libraries

* Add more details about available compilers

* Use -march=znver5 for explicitness

---------

Co-authored-by: leopekkas <leopekkas@gmail.com>

trossi committed:
Roihu HPC libraries and compiling (#2972)

* Remove outdated warning

* Update list of libraries

* Add more details about available compilers

* Use -march=znver5 for explicitness

---------

Co-authored-by: leopekkas <leopekkas@gmail.com>

leopekkas committed:
Update Roihu Slurm instructions (#2976)

* Remove SMT instructions from examples, add SBoF example

* Remove smt examples, use default mem on full nodes

leopekkas committed:
Update Roihu Slurm instructions (#2976)

* Remove SMT instructions from examples, add SBoF example

* Remove smt examples, use default mem on full nodes

leopekkas committed:
WIP: Roihu user spack tutorial (#2968)

* Typo fixes

* Add section for Spack installations in quickstart guide

* Add a warning about user-spack being WIP

leopekkas committed:
WIP: Roihu user spack tutorial (#2968)

* Typo fixes

* Add section for Spack installations in quickstart guide

* Add a warning about user-spack being WIP

mvsjober committed:
Update vLLM versions

mvsjober committed:
Update vLLM versions

jussienko committed:
Merge branch 'roihu' into connect-compute-node

leopekkas committed:
Add warning about Satama not being in use yet

leopekkas committed:
Add warning about Satama not being in use yet

jussienko committed:
Small text improvements based on review

jussienko committed:
Small text improvements based on review

leopekkas committed:
Merge branch 'master' into roihu

Iidahak committed:
Update python-data.md

Add gpu installations to python-data.md

Iidahak committed:
Update python-data.md

Add gpu installations to python-data.md

Iidahak committed:
Update python-data.md

Iidahak committed:
Update python-data.md

leopekkas committed:
Fix Roihu SSH instructions (#2978)

* Fix SSH instruction typos

* Add info about python script printing cert expiration

leopekkas committed:
Fix Roihu SSH instructions (#2978)

* Fix SSH instruction typos

* Add info about python script printing cert expiration

Iidahak committed:
Update python-data.md

Iidahak committed:
Update python-data.md

Iidahak committed:
Merge pull request #2980 from CSCfi/python-updates

Python-data updates

nruneberg committed:
Update OpenMPI version in cp2k.md

nruneberg committed:
Update OpenMPI version in cp2k.md

jussienko committed:
Merge pull request #2977 from CSCfi/connect-compute-node

Document how to connect to a compute node of running Slurm job

jussienko committed:
Include Roihu and improve the documentation

jussienko committed:
Add Roihu and improve description a bit

jussienko committed:
Include Roihu and improve a bit

jussienko committed:
Add Roihu and improve the description

mvsjober committed:
Roihu ML base container (#2981)

* Add ML base image example to Roihu getting started

* Further fixes to ML base image description

* Remove Satama warning

* Add section lines for containers

---------

Co-authored-by: leopekkas <leopekkas@gmail.com>

mvsjober committed:
Roihu ML base container (#2981)

* Add ML base image example to Roihu getting started

* Further fixes to ML base image description

* Remove Satama warning

* Add section lines for containers

---------

Co-authored-by: leopekkas <leopekkas@gmail.com>

leopekkas committed:
Update Roihu Slurm docs (#2982)

* Add a max nodes section for Roihu GPU partitions

leopekkas committed:
Update Roihu Slurm docs (#2982)

* Add a max nodes section for Roihu GPU partitions

jussienko committed:
Fix links

jussienko committed:
Apply suggestions from code review

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

jussienko committed:
Update version information for compute-sanitizer

leopekkas committed:
WIP: Fix Roihu details in docs (#2985)

* Fix Roihu documentation details

* Update allas-in-roihu.md

* Fix gpupilot partition name

---------

Co-authored-by: kkmattil <kimmo.mattila@csc.fi>

henryjuho committed:
sbf exclusive use clarification (#2991)

* --exclusive clarity

* typos

---------

Co-authored-by: Henry Barton <henry.barton@csc.fi>

Commit:
warning to avoid mixing archictures on roihu

jlento committed:
Roihu user spack tough case (#2989)

* Update roihu-user-spack.md

What to do when spack is not using upstream packages that it should

Iidahak committed:
Update tykky.md

Add documentation for uv support in Tykky

Iidahak committed:
Update tykky.md

jlento committed:
Update roihu-user-spack.md

variable was not expanded as I expected, but seems to create the source-cache directory under the environment root - fine

Iidahak committed:
Update tykky.md

Iidahak committed:
Update tykky.md

Iidahak committed:
Update tykky.md

Iidahak committed:
Update docs/computing/containers/tykky.md

Co-authored-by: Mats Sjöberg <mats.sjoberg@csc.fi>

Iidahak committed:
Update docs/computing/containers/tykky.md

Co-authored-by: Mats Sjöberg <mats.sjoberg@csc.fi>

ktiits committed:
Roihu updates to GIS tutorials (#3001)

* Roihu updates to GIS tutorials

Commit:
Replaced Mahti references to Roihu.

Commit:
Added Roihu related IP-info and removed references to Puhti.

henryjuho committed:
sbf exclusive warning added to script page (#3000)

Co-authored-by: Henry Barton <henry.barton@csc.fi>

Commit:
julia documentation for roihu

Commit:
matlab documentation for roihu

Commit:
update roihu containers

Commit:
remove compatibility

helijuottonen committed:
Add Roihu to r-env documentation

Commit:
GPU-aware MPI

Commit:
gpu-aware mpi julia

jlento committed:
Update roihu-user-spack.md (#3010)

* WIP: Update roihu-user-spack.md

* Update roihu-user-spack.md

Commit:
julia

Commit:
roihu base containers

jlento committed:
Update roihu-user-spack.md

Format fix

jlento committed:
Update roihu-user-spack.md

Commit:
roihu base images

leopekkas committed:
Roihu Slurm instruction fixes (#2987)

* Add max CPU cores to Roihu GPU partitions

* Update GPU partition CPUs into a per GPU basis

* Use per-job categorization on single node partitions

* Move interactive partitions to separate section

* Add R/W speeds of local storage

Commit:
roihu base containers

Commit:
fix gdrcopy issue with julia cuda-aware mpi

Commit:
remove env var

Commit:
roihu base containers

helijuottonen committed:
Update docs/apps/r-env.md

Co-authored-by: Mats Sjöberg <mats.sjoberg@csc.fi>

ktiits committed:
Roihu gis (#3027)

* Add what's new about Ubuntu 26.04

* fix language

* add image name

* add commas

* Add Roihu QGIS 3.44

Updated version information, usage instructions, and citation year for QGIS documentation.

* Enhance QGIS documentation with Terminal instructions

Added instructions for starting QGIS from Terminal and updated section headers.

* Add zonation 5.2.4 in Roihu

Updated available versions and usage details for Zonation.

* docs: update Ollama package

* Add SNAP 13.0 (#2898)

* Add SNAP 13.0

Updated SNAP documentation to reflect new versions and changes in available tools and usage instructions.

* Update docs/apps/snap.md

typo

Co-authored-by: EetuHuuskoCSC <116141296+EetuHuuskoCSC@users.noreply.github.com>

---------

Co-authored-by: EetuHuuskoCSC <116141296+EetuHuuskoCSC@users.noreply.github.com>

* Update systems-puhti.md (#3019)

Updated one "spring" into "summer"

Co-authored-by: Joonas Somero <50655931+joonas-somero@users.noreply.github.com>

* Update LAStools documentation for Roihu and versions

Added information about LAStools availability on Roihu and updated version details for 2023-2026. Clarified usage instructions for licensed versions and command line tools.

* Add Roihu SNAP

Updated SNAP documentation to reflect changes in available versions and usage instructions, including GUI access and Python integration.

* Add tags

* Add 'Roihu' to available web interfaces

---------

Co-authored-by: Antonio Delgado <adelgado@csc.fi>
Co-authored-by: Antonio J. Delgado <ajdelgado@users.noreply.github.com>
Co-authored-by: Alvaro Gonzalez <alvaro.gonzalez@csc.fi>
Co-authored-by: Iidahak <75577529+Iidahak@users.noreply.github.com>
Co-authored-by: Mats Sjöberg <mats.sjoberg@csc.fi>
Co-authored-by: EetuHuuskoCSC <116141296+EetuHuuskoCSC@users.noreply.github.com>
Co-authored-by: Maria Lehtivaara <maria.lehtivaara@csc.fi>
Co-authored-by: Joonas Somero <50655931+joonas-somero@users.noreply.github.com>

Commit:
Changed Puhti service to Roihu.

helijuottonen committed:
Update r-env.md

Changes based on Mats' comments + Cmdstan version update

helijuottonen committed:
Update parallel-r-examples.md

Threading: change from APPTAINERENV_OMP_NUM_THREADS to OMP_NUM_THREADS. Add caveat on thread use.

helijuottonen committed:
Update r-env.md

Correct preposition

helijuottonen committed:
Remove APPTAINERENV from threading settings for Roihu

For Roihu, removing OMP_NUM_THREADS=1 from the container and moving it to the module file. Should allow using OMP_NUM_THREADS instead of APPTAINERENV_OMP_NUM_THREADS in batch job scripts.

kkmattil committed:
Kkmattil patch 1 (#3016)

* Update and rename allas-hpc.md to allas-roihu.md

* Update allas-roihu.md

* Use s3allas

---------
Co-authored-by: leopekkas <leopekkas@gmail.com>
Co-authored-by: kkmattil <kimmo.mattila@csc.fi>

leopekkas committed:
Add link to roihu-data-preparation for rebasing

leopekkas committed:
Merge branch 'master' into roihu

Commit:
Updated Abaqus and Ansys user documentation for Roihu

sjsillan committed:
Merge pull request #3035 from CSCfi/roihu_engineering_apps

Updated Abaqus and Ansys user documentation for Roihu

leopekkas committed:
Merge branch 'master' into roihu

leopekkas committed:
Update links and info for GA (#3033)

* Update links and info for GA

* General updates, updating from Mahti/Puhti to Roihu

* Fix general Mahti/Puhti distinctions to Roihu

Commit:
Updated licensing information and Roihu related modifications

sjsillan committed:
Merge pull request #3036 from CSCfi/roihu_engineering_apps

Updated licensing information and Roihu related modifications

Commit:
Modifications realated to Roihu

sjsillan committed:
Merge pull request #3039 from CSCfi/roihu_engineering_apps

Modifications realated to Roihu

leopekkas committed:
Update projdata to dataset

leopekkas committed:
Small fixes

leopekkas committed:
Merge pull request #3040 from CSCfi/roihu-dataset

Update projdata to dataset

leopekkas committed:
Merge branch 'master' into roihu

leopekkas committed:
Add warning to direct data migration about availability

leopekkas committed:
Typo fixes in Roihu tutorials

ktiits committed:
WIP: Roihu gis2 (#3047)

* Update Orfeo ToolBox documentation for Roihu

Added Roihu as an available platform for Orfeo ToolBox and updated installation instructions for versions 9.1.1 and 8.0.1.

* Add Roihu to CloudCompare

Updated available versions of CloudCompare and clarified access through web interfaces.

* Update usage section for CloudCompare documentation

Clarified the availability of CloudCompare in the web interface.

* SAGA GIS for Roihu

Updated SAGA GIS documentation to reflect the availability of version 9.10.2 in Roihu and clarified usage instructions.

* Add Roihu to GRASS GIS 

Updated GRASS GIS documentation to include Roihu as an available web interface and clarified installation instructions.

* Shorter name

* Add Roihu versions

Added GDAL version 3.12.4 and 3.12.2 details for Roihu. Updated citation year to 2026.

* Add roihu installation specs

Added installation settings for GDAL version 3.12.2.

* Add Roihu PDAL

* Roihu R for GIS

* Roihu and LUMI to whiteboxtools page

* Add geo2ml, spyder, planetary-computer and xarry-eopf.

* Add LUMI 3.14.4 geoconda

* Roihu for ODM

* styling fix

* shorter name for tool

* Styling fix

* Add spyder link

* styling fix

* styling fix

Added instructions for preparing aerial images for OpenDroneMap.

* styling fix

* styling fix

Added a note about optimal project size and CPU usage for OpenDroneMap.

* styling fix

* styling fix

Added additional note about optimal image count and CPU usage.

* typo

leopekkas committed:
Update Roihu FAQ (#3042)

* Update Roihu FAQ

* Update external WIP links to internal ones

Added 2 Q&As from weeklies.
Fixed the accessibility of links that said "here" or similar

* Update Roihu schedules

---------

Co-authored-by: Maria Lehtivaara <maria.lehtivaara@csc.fi>

trossi committed:
Warn that email notifications are not enabled

trossi committed:
Merge pull request #3050 from CSCfi/roihu-email

Add a warning that email notifications are not enabled yet

trossi committed:
Update GPAW page for roihu

jussienko committed:
Merge pull request #3051 from CSCfi/roihu-gpaw

Update GPAW page for roihu

helijuottonen committed:
Add x86_64 to RStudio user state reset

helijuottonen committed:
Add x86_64 to RStudio user state reset

helijuottonen committed:
Merge pull request #3054 from CSCfi/RStudio_pathfix

Add x86_64 to RStudio user state reset

mvsjober committed:
Add TensorFlow, changed order also on PyTorch setting Roihu before Puhti and Mahti.

mvsjober committed:
Fix markdown error in tensorflow.md

mvsjober committed:
Merge pull request #3056 from CSCfi/roihu-tensorflow

Add TensorFlow, changed order also on PyTorch setting Roihu first

leopekkas committed:
Merge branch 'master' into roihu

leopekkas committed:
Update Roihu schedule marker

helijuottonen committed:
Update stan batch job settings

Update Cmdstan instructions: no need for any extra settings for Cmdstan batch jobs. SING_FLAGS does not need to be replaced by anything.

helijuottonen committed:
Merge pull request #3059 from CSCfi/hj_cmdstan

Update stan batch job settings

leopekkas committed:
Roihu compression tutorial (#3058)

* Roihu compression tutorial

* Remove compression tutorial

helijuottonen committed:
Update SSH tunnelling status for RStudio Server

Add mention that SSH tunneling for RStudio Server is available on Puhti and Mahti (meaning not on Roihu yet).

helijuottonen committed:
Merge pull request #3060 from CSCfi/helijuottonen-patch-1

Update SSH tunnelling status for RStudio Server

helijuottonen committed:
Removing -l option from batch job scripts

Removing -l option from the first line of batch job scripts (left there as a remnant from older times?).

helijuottonen committed:
Removing -l option from R batch job scripts

helijuottonen committed:
Merge pull request #3061 from CSCfi/helijuottonen-patch-1

Removing -l option from R batch job scripts

juhanikataja committed:
elmer on roihu-cpu

juhanikataja committed:
Merge pull request #3063 from juhanikataja/roihu

elmer on roihu-cpu

sjsillan committed:
Minor change in comsol.md

Fixed Puhti web interface --> Roihu web interface.

leopekkas committed:
Roihu tmux instructions (#3064)

* Update docs to use tmux instead of screen in Roihu

leopekkas committed:
Add Roihu schedule to data migration guide

nruneberg committed:
Gaussian on Roihu

rkronberg committed:
Roihu accounts (#2902)

* Roihu updates to Accounts

* Clarify strong identification

* Update billing.md

* Update how-to-add-service-access-for-project.md

* Update how-to-increase-disk-quotas.md

* Update how-to-manage-your-project.md

* Update how-to-increase-disk-quotas.md

* Create how-to-access-roihu-large-partition.md

* Update strong identification and LoA section

* Add preliminary dataset instructions

* Remove Mahti large access from main index

* Update large partition access docs

---------

Co-authored-by: Kylli Ek <kylli.ek@csc.fi>
Co-authored-by: leopekkas <leopekkas@gmail.com>
Co-authored-by: Sami Salonen <63836095+samisalonen-csc@users.noreply.github.com>
Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

nruneberg committed:
Merge pull request #3069 from CSCfi/roihu_gaussian

Gaussian on Roihu

leopekkas committed:
Roihu GA (#3065)

* Update FAQ for GA

* Update Roihu tutorials for post-GA

* Move from future to present tense in Roihu docs

leopekkas committed:
Fix elmer availability tag

leopekkas committed:
Add instructions for adding Roihu as a service

Iidahak committed:
Minor fixes to python docs

Iidahak committed:
add Mahti and Puhti specification

lvarin commented on docs/cloud/dbaas/mariadb-accessing.md:
````diff
@@ -135,24 +135,20 @@ ERROR 1044 (42000): Access denied for user 'username'@'%' to database 'databasen
 Either the database specified does not exist, or the username specified has no access to it.
 
 
-```
-mariadb-dump: Got error: 2026: "TLS/SSL error: self-signed certificate" when trying to connect.
-```
-
-The `--ssl` flag alone isn't enough, you need to supply the client `--ssl-verify-server-cert=OFF` to skip certificate verification.
-
````
This should not be deleted

leopekkas replied:
Fixed

leopekkas committed:
Fix based on review

leopekkas committed:
Merge branch 'master' into roihu

nruneberg committed:
Add Molpro documentation for Roihu

leopekkas committed:
Add hyperqueue availability for roihu

leopekkas committed:
Update roihu getting started for ga (#3070)

* Update roihu general tutorial for GA

leopekkas committed:
Merge branch 'master' into roihu

Commit:
Added FirecREST HPC API documentation

Commit:
document hyperqueue for roihu

Iidahak committed:
update dask-python.md to Roihu

Commit:
improved hyperqueu documentation

Commit:
fix hyperqueue links

leopekkas committed:
Fix Roihu large partition link

leopekkas committed:
Merge branch 'master' into roihu

mvsjober committed:
Add JAX for Roihu

mvsjober committed:
Merge pull request #3075 from CSCfi/roihu-jax

Add JAX for Roihu

Commit:
refactorings to throughput page

Commit:
fix grammar

Commit:
define large

leopekkas committed:
Merge branch 'master' into roihu

Commit:
fixes

Commit:
Merge branch 'roihu' of github.com:CSCfi/csc-user-guide into roihu

mvsjober committed:
Updated ML guide front page, and GPU page to Roihu age

Commit:
refactoring

mvsjober committed:
Merge pull request #3078 from CSCfi/ml-guide-roihu-updates

Updated ML guide front page, and GPU page to Roihu age

Iidahak committed:
Add JAX ML framework to Python.md

Iidahak committed:
Fix wording in dask-python.md

Iidahak committed:
Update Dask-python.md

Commit:
refactoring

jlento committed:
Create cdo.md (#3067)

* Create cdo.md

* Add licensing for cdo

---------

Co-authored-by: leopekkas <leopekkas@gmail.com>

Commit:
integrated snakemake tutorial into snakemake application page

Commit:
Merge branch 'roihu' of github.com:CSCfi/csc-user-guide into roihu

Commit:
integrate nextflow tutorial into nextflow application page

Commit:
move fireworks to application page

leopekkas committed:
Remove pilot specific sections in the docs (#3045)

* Remove pilot specific sections in the docs

* Remove pilot project specific slurm instructions

* Change general Roihu disclaimer label for GA

* Remove pilot SSH key fingerprints by GA

Iidahak committed:
update dask-python.md

Commit:
improved throughput page

Commit:
Merge branch 'roihu' of github.com:CSCfi/csc-user-guide into roihu

Iidahak committed:
Merge pull request #3079 from CSCfi/roihu-python

Update python docs and dask-python.md

Commit:
refactoring

Commit:
Merge branch 'roihu' of github.com:CSCfi/csc-user-guide into roihu

nruneberg committed:
Add F12 benchmarks and memory guidance to Molpro Roihu docs

nruneberg committed:
Merge pull request #3080 from CSCfi/roihu_molpro

Add Molpro documentation for Roihu

Commit:
links

Commit:
Merge branch 'roihu' of github.com:CSCfi/csc-user-guide into roihu

leopekkas committed:
Fix comsol list formatting

leopekkas committed:
Fix typos

leopekkas committed:
General fixes and typos

leopekkas committed:
Fix broken link

leopekkas committed:
Fix internal link

Commit:
removed duplicated high-throughtput details from array jobs page

leopekkas committed:
Merge branch 'master' into roihu

robinkar committed:
Add Roihu web interface (#3066)

* Update docs for www-Roihu

* Add short docs about marimo

* Update OOD screenshot and alt text

* Fix missing Roihu mentions in OOD docs

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

* Update www-Roihu screenshot

* Add COMSOL to www-Roihu

---------

Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

leopekkas committed:
Update docs wording for GA, small fixes

nruneberg committed:
Fix Allas URL for ORCA example input

nruneberg committed:
Merge pull request #3083 from CSCfi/roihu_orca

Add ORCA documentation for Roihu

galfthan committed:
Roihu pricing update (#3072)

* Initial roihu pricing description

* Tuned texts, and removed SBOF term

* Add corresponding roihu disk section

* Add a top-level explanation of BUs, clarify text

---------

Co-authored-by: leopekkas <leopekkas@gmail.com>
Co-authored-by: Henry Barton <henry.barton@csc.fi>
Co-authored-by: Leopekka Saraste <45951298+leopekkas@users.noreply.github.com>

leopekkas committed:
Remove deprecated warning about LUE on Roihu

leopekkas committed:
Fix deprecated LUE schedule

Commit:
throughput

Commit:
refactoring to gnu parallel page

nruneberg committed:
Add TURBOMOLE documentation for Roihu with performance benchmarks

nruneberg committed:
Merge pull request #3084 from CSCfi/roihu_turbomole

Add TURBOMOLE documentation for Roihu

Commit:
improve high-throughput page

Commit:
Merge branch 'roihu' of github.com:CSCfi/csc-user-guide into roihu

galfthan committed:
Minor edits

galfthan committed:
Merge pull request #3073 from CSCfi/roihu-firecrest

Add Roihu FirecREST HPC API documentation

galfthan committed:
Update download link to point to the releases (#3085)
